server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/manager.ftl

```diff
@@ -23,9 +23,12 @@
           <h3>Manager Server Overview</h3>
        </div>
     </div>
-    <div id="managerBanner" style="display: none;">
+    <div id="managerRunningBanner" style="display: none;">
         <div class="alert alert-danger" role="alert">Manager Server Not Running</div>
     </div>
+    <div id="managerStateBanner" style="display: none;">
+        <div class="alert alert-warning manager-banner-message" role="alert"></div>
```

Comment:
This is a unique item, and should thus be an ID rather than a CLASS.
```suggestion
        <div id="manager-banner-message" class="alert alert-warning" role="alert"></div>
```